include/gridtools/stencil_composition/sid/synthetic.hpp

```diff
@@ -73,12 +96,14 @@ namespace gridtools {
             template <>
             struct synthetic<> {
                 template <property Property, class T>
-                GT_CONSTEXPR synthetic<unique_mixin<Property, T>> set() const &&noexcept {
+                GT_CONSTEXPR synthetic<unique_mixin<Property, T>> set(
+                    property_constant<Property> = {}, meta::lazy::id<T> = {}) const &&noexcept {
                     return synthetic{};
                 }
 
                 template <property Property, class T>
-                GT_CONSTEXPR synthetic<unique_mixin<Property, decay_t<T>>> set(T &&val) const &&noexcept {
+                GT_CONSTEXPR synthetic<unique_mixin<Property, decay_t<T>>> set(
+                    T &&val, property_constant<Property> = {}) const &&noexcept {
```

Comment:
I find it quite confusing that for some properties, the `property_constant` comes first and for others it comes second. I guess this is due to the default values, but I think a uniform signature might be more important here.

Reply:
Is it better now?

Reply:
Yes, I like it better that way!